src/hb-ot-var-gvar-table.hh

```diff
@@ -42,12 +42,14 @@ namespace OT {
 
 struct contour_point_t
 {
-  void init (float x_=0.f, float y_=0.f) { flag = 0; x = x_; y = y_; }
+  void init (float x_ = 0.f, float y_ = 0.f, bool is_end_point_ = false)
+  { flag = 0; x = x_; y = y_; is_end_point = is_end_point_; }
```

Comment:
This could go without addition of `is_end_point` here IIRC, will try to do that before or after the merge.